scripts/blocklist-skylink.sh

```diff
@@ -32,21 +37,23 @@ fi
 #########################################################################
 # iterate through all servers, block the skylinks and purge it from cache
 #########################################################################
-declare -a servers=(  "eu-ger-1.siasky.net" "eu-ger-2.siasky.net" "eu-ger-3.siasky.net" "eu-ger-4.siasky.net" "eu-ger-5.siasky.net" "eu-ger-6.siasky.net" "eu-ger-7.siasky.net" "eu-ger-8siasky.net"
+declare -a servers=(  "eu-ger-1.siasky.net" "eu-ger-2.siasky.net" "eu-ger-3.siasky.net" "eu-ger-4.siasky.net" "eu-ger-5.siasky.net" "eu-ger-6.siasky.net" "eu-ger-7.siasky.net" "eu-ger-8.siasky.net"
                       "eu-fin-1.siasky.net" "eu-fin-2.siasky.net" "eu-fin-3.siasky.net" "eu-fin-4.siasky.net"
                       "eu-pol-1.siasky.net" "eu-pol-2.siasky.net" "eu-pol-3.siasky.net"
                       "us-or-1.siasky.net" "us-or-2.siasky.net"
                       "us-pa-1.siasky.net" "us-pa-2.siasky.net"
                       "us-va-1.siasky.net" "us-va-2.siasky.net" "us-va-3.siasky.net"
                       "as-hk-1.siasky.net"
-                      "siasky.xyz" "siasky.dev")
+                      "siasky.xyz" "dev1.siasky.dev" "dev2.siasky.dev" "dev3.siasky.dev")
 for server in "${servers[@]}";
 do
     for skylink in "${skylinks[@]}";
     do
         echo ".. ⌁ Blocking skylink ${skylink} on ${server}"
-
-        ssh -q -t user@${server} "docker exec sia siac skynet blocklist add $skylink && docker exec nginx curl -s -i -X PURGE http://localhost/$skylink | egrep \"^(OK|HTTP|X-)\""
+        cached_files_command="find /data/nginx/cache/ -type f | xargs --no-run-if-empty grep -Elsq '^Skynet-Skylink: ${skylink}'"
```

Comment:
is `q` searching just for first file and it stops or it just searches files until it finds a match but still searches all files?

Reply:
```suggestion
        cached_files_command="find /data/nginx/cache/ -type f | xargs -r grep -Elsq '^Skynet-Skylink: ${skylink}'"
```

shorter syntax

Reply:
> is `q` searching just for first file and it stops or it just searches files until it finds a match but still searches all files?

q was to suppress the output. Running it manually I didn't want to see all the files that were being grepped.

Reply:
> shorter syntax

added